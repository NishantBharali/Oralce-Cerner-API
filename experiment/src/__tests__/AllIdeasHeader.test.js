import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';
import { configure ,shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import "core-js/stable";
import "regenerator-runtime/runtime";
import AllIdeaHeader from "../containers/Headers and Footers/AllIdeasHeader";


configure({ adapter: new Adapter() });


it("check for test cases",()=> {
    const renderer = new ShallowRenderer();
    renderer.render(<AllIdeaHeader />);
    expect(renderer).toMatchSnapshot();

})
