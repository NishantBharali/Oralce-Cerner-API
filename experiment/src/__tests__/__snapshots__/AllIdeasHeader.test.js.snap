// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check for test cases 1`] = `
ReactShallowRenderer {
  "_context": Object {},
  "_didScheduleRenderPhaseUpdate": false,
  "_dispatcher": Object {
    "readContext": [Function],
    "useCallback": [Function],
    "useContext": [Function],
    "useDebugValue": [Function],
    "useDeferredValue": [Function],
    "useEffect": [Function],
    "useId": [Function],
    "useImperativeHandle": [Function],
    "useInsertionEffect": [Function],
    "useLayoutEffect": [Function],
    "useMemo": [Function],
    "useReducer": [Function],
    "useRef": [Function],
    "useResponder": [Function],
    "useState": [Function],
    "useSyncExternalStore": [Function],
    "useTransition": [Function],
  },
  "_element": <AllIdeaHeader />,
  "_firstWorkInProgressHook": null,
  "_forcedUpdate": false,
  "_idCounter": 0,
  "_instance": null,
  "_isReRender": false,
  "_newState": null,
  "_numberOfReRenders": 0,
  "_renderPhaseUpdates": null,
  "_rendered": <div
    className="ui pointing menu"
    style={
      Object {
        "backgroundColor": "black",
        "left": 0,
        "position": "fixed",
        "right": 0,
        "top": 0,
        "zIndex": 999,
      }
    }
  >
    <a
      className="active item"
      style={
        Object {
          "color": "white",
        }
      }
    >
      <b>
        You are viewing other ideas available in the Repository
      </b>
    </a>
    <div
      className="right menu"
    >
      <DropdownForViewAll />
      <p>
                       
      </p>
    </div>
  </div>,
  "_rendering": false,
  "_updater": Updater {
    "_callbacks": Array [],
    "_renderer": [Circular],
  },
  "_workInProgressHook": null,
}
`;
